README.md

```diff
@@ -62,6 +62,18 @@ You can deploy NGINX Kubernetes Gateway on an existing Kubernetes 1.16+ cluster.
    kubectl apply -k "github.com/kubernetes-sigs/gateway-api/config/crd?ref=v0.4.2" 
    ```
 
+1. Create the nginx-gateway namespace:
+   
+    ```
+    kubectl apply -f deploy/manifests/namespace.yaml
```

Comment:
I had to separate out the namespace from the deployment manifest so I could create the configmap in the next step.